src/pyhf/optimize/opt_scipy.py

```diff
@@ -8,11 +8,15 @@ class scipy_optimizer(object):
     def __init__(self, **kwargs):
         self.maxiter = kwargs.get('maxiter', 100000)
 
-    def unconstrained_bestfit(self, objective, data, pdf, init_pars, par_bounds):
-        # The Global Fit
+    def minimize(self, objective, data, pdf, init_pars, par_bounds, fixed_vals=None):
+        fixed_vals = fixed_vals or []
```

Comment:
`fixed_vals=[]` instead of `None` by default. Then no need for this line.

Reply:
see below